internals/cli/cmd_run.go

```diff
@@ -31,16 +31,16 @@ import (
 	"github.com/canonical/pebble/internals/systemd"
 )
 
-var shortRunHelp = "Run the pebble environment"
+var shortRunHelp = "Run the daemon"
 var longRunHelp = `
-The run command starts pebble and runs the configured environment.
+The run command starts the daemon and runs the configured environment.
 
 Additional arguments may be provided to the service command with the --args option, which
-must be terminated with ";" unless there are no further Pebble options.  These arguments
+must be terminated with ";" unless there are no further program options.  These arguments
 are appended to the end of the service command, and replace any default arguments defined
 in the service plan. For example:
 
-    $ pebble run --args myservice --port 8080 \; --hold
+    run --args myservice --port 8080 \; --hold
```

Comment:
Yeah, I agree that the examples currently don't feel clear. I like `<binary>` because it's short, but it might be a bit confusing with "binary numbers", especially for `<binary> add`. Possibly `<program>` or `<executable>` or even `<pebble>`?

Reply:
I think `<pebble>` would be the fittest. However, in Termus for instance, there are absolutely no references to `pebble` in any of the help strings (that's the point of this PR!), so for some people outside Canonical the `pebble` reference could be really confusing.

I'd love to hear @niemeyer's opinion on this matter 👍

Reply:
IMO <pebble> could be misleading but I think we could be vocal about Pebble's personality here with something like <pebble_personality> or <pebble_child> etc... A single word is always better looking but not sure if it's enough to avoid ambiguity.